decidim-core/spec/system/authentication_spec.rb

```diff
@@ -600,6 +600,16 @@
           expect(page).to have_content("Successfully")
           expect(page).to have_content(user.name)
         end
+
+        context "when admin password is expired" do
+          let(:user) { create(:user, :confirmed, :admin, password_updated_at: 91.days.ago, organization: organization) }
+
```

Comment:
We should stub the `admin_password_expiration_days` configuration accessor, as we are having a precondition here (admin_password_expiration_days is 90) but in the future we could change the days needed for this configuration to kick-in:
```suggestion

          before do
            allow(Decidim.config).to receive(:admin_password_expiration_days).and_return(90)
          end

```